docs/source/user/installing.rst

```diff
@@ -17,60 +17,192 @@ Our supported platforms are:
 * NVIDIA GPUs of compute capability 2.0 and later
 * AMD ROC dGPUs (linux only and not for AMD Carrizo or Kaveri APU)
 
+:ref:`numba-parallel` is only available on 64-bit platforms,
+and is not supported in Python 2.7 on Windows.
 
-Installing using Conda
-----------------------
+Installing using conda on x86/x86_64/POWER Platforms
+----------------------------------------------------
 
-The easiest way to install numba and get updates is by using Conda,
+The easiest way to install Numba and get updates is by using ``conda``,
 a cross-platform package manager and software distribution maintained
 by Anaconda, Inc.  You can either use `Anaconda
 <https://www.anaconda.com/download>`_ to get the full stack in one download,
 or `Miniconda <https://conda.io/miniconda.html>`_ which will install
-the minimum packages needed to get started.
+the minimum packages required for a conda environment.
 
 Once you have conda installed, just type::
 
-   $ conda install numba
+    $ conda install numba
 
 or::
 
-   $ conda update numba
+    $ conda update numba
+
+Note that Numba, like Anaconda, only supports PPC in 64-bit little-endian mode.
+
+To enable CUDA GPU support for Numba, install the latest `graphics drivers from
+NVIDIA <https://www.nvidia.com/Download/index.aspx>`_ for your platform.
+(Note that the open source Nouveau drivers shipped by default with Ubuntu do
+not support CUDA.).  Then install the ``cudatoolkit`` package::
+
+    $ conda install cudatoolkit
+
+You do not need to install the CUDA SDK from NVIDIA.
+
+
+Installing using pip on x86/x86_64 Platforms
+--------------------------------------------
+
+Binary wheels for Windows, Mac, and Linux are also available from `PyPI
+<https://pypi.org/project/numba/>`_.  You can install Numba using ``pip``::
+
+    $ pip install numba
+
+This will download all of the needed dependencies as well.  You do not need to
+have *LLVM* installed to use Numba (in fact, Numba will ignore all LLVM
+versions installed on the system) as the required components are bundled into
+the llvmlite wheel.
+
+To use CUDA with Numba installed by `pip`, you need to install the `CUDA SDK
+<https://developer.nvidia.com/cuda-downloads>`_ from NVIDIA.  Then you may need to
+set the following environment variables so Numba can locate the required
+libraries:
+
+  * ``NUMBAPRO_CUDA_DRIVER`` - Path to the CUDA driver shared library file
```

Comment:
Due to the leading indent here, it renders slightly weird, in a block and with bold fonts.